src/components/alert/alert.stories.js

```diff
@@ -0,0 +1,139 @@
+import { html } from "lit-html";
+import "../button";
+import "../alert";
+import styles from "./styles.scss";
+import { stylesheet } from "../../../.storybook/utils.js";
+
+export default {
+  title: "custom-elements/vl-alert",
+  decorators: [(story) => html`${stylesheet(styles)}${story()}`],
+  args: {
+    title: "Opgelet!",
+    icon: "warning",
+    size: "normal",
+    type: "info",
+    closable: false,
+  },
+  argTypes: {
+    title: {
+      name: "data-vl-title",
+      type: { summary: "string" },
+      description: "Attribuut wordt gebruikt om de titel te bepalen.",
+      table: {
+        defaultValue: { summary: "" },
+      },
+    },
+    icon: {
+      name: "data-vl-icon",
+      type: "select",
+      options: ["warning", "check", "info-circle"],
+      description: "Attribuut wordt gebruikt om het icoon type te bepalen.",
+      table: {
+        type: { summary: "string" },
+        defaultValue: { summary: "" },
+      },
+    },
+    size: {
+      name: "data-vl-size",
+      type: "select",
+      options: ["normal", "small"],
```

Comment:
Hoe kom je aan die normal? Als ik naar de component kijk lijkt hij enkel `small` te begrijpen.

Reply:
Dat is de default... werkt normaal zoals het moet hoor.

Reply:
Aangepast.